test/unit-test/twcc_manager/twcc_manager_utest.c

```diff
@@ -176,22 +462,223 @@ void test_twccFindPacket_Found( void )
+        TEST_ASSERT_EQUAL( RTCP_TWCC_MANAGER_RESULT_OK,
+                           result );
+
+        arrivalInfoList[i].seqNum = packetInfo.packetSeqNum;
+        if( i % 2 == 0 ) {
+            arrivalInfoList[i].remoteArrivalTime = RTCP_TWCC_PACKET_LOST_TIME; // Set some packets as lost
+        }
+        else {
+            arrivalInfoList[i].remoteArrivalTime = localSentTime + ( int64_t )i; // Set some packets as received
+        }
+    }
+
+    // Creating a valid RtcpTwccPacket_t
+    twccPacket.arrivalInfoListLength = TWCC_PACKET_INFO_ARRAY_LENGTH;
+    twccPacket.pArrivalInfoList = arrivalInfoList;
+
+    result = RtcpTwccManager_HandleTwccPacket( &( twccManager ),
+                                               &( twccPacket ),
+                                               &bandwidthInfo );
+
+    TEST_ASSERT_EQUAL( RTCP_TWCC_MANAGER_RESULT_OK,
+                       result );
+    TEST_ASSERT_GREATER_THAN( 0,
+                              bandwidthInfo.duration );
+    TEST_ASSERT_EQUAL( TWCC_PACKET_INFO_ARRAY_LENGTH,
+                       bandwidthInfo.sentPackets );
+    TEST_ASSERT_GREATER_THAN( 0,
+                              bandwidthInfo.sentBytes );
+    TEST_ASSERT_GREATER_THAN( 0,
+                              bandwidthInfo.receivedBytes );
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Validate Twcc Manager Handle packet functionality when unidentified Sequence Number Packet is found.
+ */
+void test_twccHandlePacket_unidentifiedSeqNum( void )
+{
+    RtcpTwccManager_t twccManager;
+    RtcpTwccManagerResult_t result;
+    RtcpTwccPacket_t twccPacket;
+    TwccBandwidthInfo_t bandwidthInfo;
+    TwccPacketInfo_t packetInfo;
+    PacketArrivalInfo_t arrivalInfo;
+    uint64_t localSentTime = time( NULL );
+    uint16_t seqNum = 256;
+    size_t i;
+
+    result = RtcpTwccManager_Init( &twccManager,
+                                   &twccPacketInfoArray[0],
+                                   TWCC_PACKET_INFO_ARRAY_LENGTH );
+    TEST_ASSERT_EQUAL( RTCP_TWCC_MANAGER_RESULT_OK,
+                       result );
+
+    for(i = 0; i < TWCC_PACKET_INFO_ARRAY_LENGTH; i++)
+    {
+        packetInfo.packetSize = i;
+        packetInfo.localSentTime = localSentTime + ( int64_t )i;
+        packetInfo.packetSeqNum = seqNum + ( uint16_t )i;
+        result = RtcpTwccManager_AddPacketInfo( &twccManager,
+                                                &packetInfo );
+        TEST_ASSERT_EQUAL( RTCP_TWCC_MANAGER_RESULT_OK,
+                           result );
+    }
+
+    // Create a RtcpTwccPacket_t with a sequence number that doesn't exist in the RtcpTwccManager_t
+    twccPacket.arrivalInfoListLength = 1;
+    arrivalInfo.seqNum = seqNum + TWCC_PACKET_INFO_ARRAY_LENGTH;
+    arrivalInfo.remoteArrivalTime = 0;
+    twccPacket.pArrivalInfoList = &arrivalInfo;
+
+    RtcpTwccManager_HandleTwccPacket( &twccManager,
```

Comment:
```suggestion
    result = RtcpTwccManager_HandleTwccPacket( &twccManager,
```

Reply:
Thanks for pointing this out, resolved it. will raise another PR for the changes.